exercises/luhn/LuhnExample.m

```diff
@@ -0,0 +1,50 @@
+#import "LuhnExample.h"
+
+@implementation Luhn
+
+- (instancetype)initWithString:(NSString *)string {
+    self = [super init];
+    
+    if (self) {
+        _isValid = [Luhn validateString:string];
+    }
+    
+    return self;
+}
+
++ (BOOL)validateString:(NSString *)string {
+    string = [string stringByReplacingOccurrencesOfString:@" " withString:@""];
```

Comment:
If there was an equivalent method that mutated the string in place, I'd use that, but since there's not I think I'll just disable the warning, rather than creating another variable.